src/utils/ruby-paths.js

```diff
@@ -0,0 +1,11 @@
+export function rubyGemDownloadRecipe(parsedFile) {
```

Comment:
note that a drawback of this is that in order to avoid circular dependencies, the param and return type cannot be typed :(